app/controllers/content.js

```diff
@@ -7,11 +7,12 @@ import permissions from 'ember-osf/const/permissions';
 /**
  * Takes an object with query parameter name as the key and value, or [value, maxLength] as the values.
  *
+ * @method queryStringify
  * @param queryParams {!object}
  * @param queryParams.key {!array|!string}
  * @param queryParams.key[0] {!string}
  * @param queryParams.key[1] {int}
- * @returns {string}
+ * @return {string}
```

Comment:
Not requesting a change in scope of this PR, but as a general note this docstring isn't quite clear- it describes the inputs but not the intended output or behavior. In particular, how this differs from `$.param`. (since Ember bundles jquery by default)